Makefile

```diff
@@ -13,7 +13,8 @@ build: $(THEME)
 
 .PHONY: update
 update: $(THEME)
```

Comment:
Yep. I agree. Just one question am I supposed to type `make theme/gitea`? Maybe we should simply use `make theme` or keep update and delete theme/gitea.